include/h_malloc.h

```diff
@@ -15,6 +15,11 @@ extern "C" {
 #define h_realloc realloc
 #define h_aligned_alloc aligned_alloc
 #define h_free free
+#if CONFIG_BLOCK_OPS_CHECK_SIZE && !defined(HAS_ARM_MTE)
```

Comment:
Maybe also cover the wide variants `wmemset(3)`, `wmemcpy(3)` and `wmemmove(3)`?